deploy/examples/create-external-cluster-resources.py

```diff
@@ -1821,6 +1913,35 @@ def gen_json_out(self):
                     }
                 )
 
+        # if 'TOPOLOGY_POOLS', 'TOPOLOGY_FAILURE_DOMAIN_LABEL', 'TOPOLOGY_FAILURE_DOMAIN_VALUES'  exists,
+        # then only add 'topology' StorageClass
+        if (
+            self.out_map["TOPOLOGY_POOLS"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_LABEL"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_VALUES"]
+        ):
+            json_out.append(
+                {
+                    "name": "ceph-rbd-topology",
+                    "kind": "StorageClass",
+                    "data": {
+                        "topologyFailureDomainLabel": self.out_map[
```

Comment:
To follow up from [this comment](https://github.com/rook/rook/pull/13821#discussion_r1523666839), the format currently here in this PR is invalid for the storage class, right? The topology needs to be formatted differently, so it seems we need to change the format or else it won't work.

Reply:
it isn't an invalid ocs operator will keep the change